akka-stream-tests/src/test/scala/akka/stream/scaladsl/SinkForeachParallelSpec.scala

```diff
@@ -82,13 +82,11 @@ class SinkForeachParallelSpec extends AkkaSpec {
           Await.ready(latch, 10.seconds)
         }
       }).withAttributes(supervisionStrategy(resumingDecider)))
-      p.onFailure { case e ⇒ assert(e.getMessage.equals("err1")); Unit }
```

Comment:
This is a non-sense assert as this would be executed and thrown in some other thread.